### DISCUSSION TITLE
Scene Matching (Online Episodes) with GEVI Scene ?

### DISCUSSION THREAD

**Discussion by Davijon31** [CodyBerenson/PGMA-Modernized#274] [Q&A]
Hi, 

I'm new here and first of all, thank you for your hard work.

I've successfully matched my films collection with Gevi agent.
But I can't find the way to match online episodes with GeviScene agent.

Could you explain me what I'm doing wrong ?
Here some examples of the titles I try to match :

(Out In Public) - Anal Sex Court Side (2013).mp4
https://gayeroticvideoindex.com/episode/687

(Czech Hunter) - CzechHunter #84 (2013).mp4
https://gayeroticvideoindex.com/episode/1188

(Cocky Boys) - Austin Wolf & Tannor Reed (2020).mp4
https://gayeroticvideoindex.com/episode/300026

(BelAmi) - Last Summer in Greece - Part 9 - Adam and Hoyt (2016).mp4
https://gayeroticvideoindex.com/episode/15494

(BelAmi) - Documentary Bite my Ass (2016).mp4
https://gayeroticvideoindex.com/episode/15653

(BelAmi) - Bobby and Joel - Condom Archive (2016).mp4
https://gayeroticvideoindex.com/episode/15873

I've read that we may add the scene number, so I've tried for this title, but with no success neither :

(NakedSword) - The Chosen Few (2018) - {Scene 4}.mp4
https://gayeroticvideoindex.com/video/61257

There is probably something I misunderstood, and I would appreciate your support.

Thanks for your help

**Comment by JPH71** [comment]
GeviScenes works a bit differently from the other agents as the website
does not allow searches for scenes...

This is how you name them for example:  (BelAmi) - Bobby and Joel - Condom
Archive (2016).mp4 - https://gayeroticvideoindex.com/episode/15873
should be renamed as (BelAmi) - Bobby and Joel - Condom Archive {15873N}
(2016).mp4

The N after the number means do not rotate the episode picture, L should be
used to Rotate Left 90 Degrees and R for rotate Right 90 degrees....

GEVIScenes only matches videos with episode in the url.... if it has video
use GEVI

Let me know if you have any further problems!


Jason x

**Reply by Zarcal79** [reply]
sorry for butting in on a thread but I am failing to match an online scene title 
Cockyboys: Fans Only With Boomer Banks and Adam Ramzi (2018)
 Ive found on GEVI and IAFD. I think the issue is the : in the title. I thought I had read how to resolve this previously but I am unable to find it.

https://gayeroticvideoindex.com/episode/300346 and https://www.iafd.com/title.rme/title=cockyboys%3a+fans+only+with+boomer+banks+and+adam+ramzi/year=2018/cockyboys%3a-fans-only-with-boomer-banks-and-adam-ramzi.htm

file name Im using is (CockyBoys) - Fans Only with Boomer Banks and Adam Ramzi {300346N} (2018) - [Boomer Banks, Adam Ramzi]

**Comment by JPH71** [comment]
@Zarcal79 - please do not apologise for asking for help.
You left out a space-dash-space in your title - it should be named like
this  (CockyBoys) - Fans Only! with Boomer Banks and Adam Ramzi - {300346N}
(2018) - [Boomer Banks, Adam Ramzi]

Cheers
Jason x

**Reply by Zarcal79** [reply]
Thanks for the reply
Its still failing got this from the GEVIscene log
GEVIScenes  - UTILS :: GEVIScenes Title              CockyBoys: Fans Only! with Boomer Banks & Adam Ramzi
GEVIScenes  - UTILS :: Comparison Title              aaaaaabbbccddefhiikklmmmnnnnooooorrssstwyyyz
GEVIScenes  - UTILS :: File Title                    Fans Only! with Boomer Banks and Adam Ramzi
GEVIScenes  - UTILS :: Comparison Title List         ['aaaaaabbddefhiiklmmmnnnnooorrsstwyz']
GEVIScenes  - UTILS :: Title Comparison Test         Failed
GEVIScenes - SEARCH:: ❌ File: __init__.py, Line: 120, Function: _process_function_call -> _search -> search
GEVIScenes - SEARCH::    Error getting Site Title: < Title Match Failure! >

**Comment by JPH71** [comment]
Delete the log file
then, rescrape
send the log file again.....
With all errors from now on send the whole log file... then I can see the
whole chain of events that lead to the error...

Cheers Jason

**Reply by Zarcal79** [reply]
I managed to get it to work but have attached 2 log files anyway - one failed and one successful.
GEVI had the studio name and a : in the title. changed the file name to match but used a space - space instead of the :
So the full file name (CockyBoys) - Cockboys - Fans Only! with Boomer Banks and Adam Ramzi {300346N} (2018) - [Boomer Banks, Adam Ramzi] this resulted in a match

Many thanks for the help and for a great agent. I lost my entire collection but now I can rebuild it with all the info etc :)

[com.plexapp.agents.GEVIScenes_FAILED.log](https://github.com/CodyBerenson/PGMA-Modernized/files/12386689/com.plexapp.agents.GEVIScenes_FAILED.log)
[com.plexapp.agents.GEVIScenes_Successful.log](https://github.com/CodyBerenson/PGMA-Modernized/files/12386690/com.plexapp.agents.GEVIScenes_Successful.log)

**Reply by JPH71** [reply]
This was because the scene has been named incorrectly on the site....

**Comment by JPH71** [comment]
Brilliant – it worked out....If you need any help please feels free to ask...Make sure that if it is a matching issue or an error that you raise it as an Issue rather than a discussion – as I tend to look at the issues more than the discussions when I am going through a back log of errors.. Cheers Jason xSent from Mail for Windows From: Zarcal79Sent: 19 August 2023 22:52To: CodyBerenson/PGMA-ModernizedCc: JPH71; CommentSubject: Re: [CodyBerenson/PGMA-Modernized] Scene Matching (Online Episodes) with GEVI Scene ? (Discussion #274) I managed to get it to work but have attached 2 log files anyway - one failed and one successful.GEVI had the studio name and a : in the title. changed the file name to match but used a space - space instead of the :So the full file name (CockyBoys) - Cockboys - Fans Only! with Boomer Banks and Adam Ramzi {300346N} (2018) - [Boomer Banks, Adam Ramzi] this resulted in a matchMany thanks for the help and for a great agent. I lost my entire collection but now I can rebuild it with all the info etc :)com.plexapp.agents.GEVIScenes_FAILED.logcom.plexapp.agents.GEVIScenes_Successful.log—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>
